### PR TITLE
Fix moving route ip to bridge test 

### PR DESCRIPTION
**Pull request by qinqon**
It should fix the moving defout route ip to bridge test

### CODE REVIEW

File: test/e2e/default_bridged_network_test.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ var _ = Describe("NodeNetworkConfigurationPolicy default bridged network", func(
 `)
 	// FIXME: This is a pending spec since we have to discover why the
 	//        cluster never goes back at kubevirtci provider
-	XContext("when there is a default interface with dynamic address", func() {
+	Context("when there is a default interface with dynamic address", func() {
```
Comment (fedepaol) [inline test/e2e/default_bridged_network_test.go]: If the spec is now working the comment should be removed?

**Comment by phoracek** [comment]
@qinqon do you want to change to k8s-1.15.1 now? Would need some automation/ renaming and stuff.

File: test/e2e/default_bridged_network_test.go (hunk 1 above)
Comment (qinqon) [reply test/e2e/default_bridged_network_test.go]: Nah it's not working :-/

**Comment by qinqon** [comment]
@phoracek just updating kubevirtci here to see if my local results are same at stdci

File: test/e2e/default_bridged_network_test.go (hunk 1 above)
Comment (qinqon) [reply test/e2e/default_bridged_network_test.go]: @fedepaol now should work let me remove it.

**Comment by phoracek** [comment]
@qinqon looks like it fails only due to focus, the test has passed

File: test/e2e/default_bridged_network_test.go [hunk 2]
```diff
@@ -81,17 +83,14 @@ var _ = Describe("NodeNetworkConfigurationPolicy default bridged network", func(
...
 
-			By("Waiting until the node becomes ready again")
-			waitForNodesReady()
-
 			By("Remove the policy")
 			deletePolicy("default-network")
 		})
 
-		It("should successfully move default IP address on top of the bridge", func() {
+		FIt("should successfully move default IP address on top of the bridge", func() {
```
Comment (qinqon) [inline test/e2e/default_bridged_network_test.go]: Remove focus

**Comment by phoracek** [comment]
OpenShift timed out on tests initialization @qinqon 

```
*** Test killed with quit: ran too long (10m0s).
```

**Comment by phoracek** [comment]
@SchSeba it did not time out on tests. It stuck before any test was executed. That has to be resolved before we merge.

**Comment by qinqon** [comment]
Locally is working fine with os-3.11.0, have added ginkgo verbose just to stdci.

**Comment by qinqon** [comment]
ci test please

**Comment by qinqon** [comment]
k8s has just fail eth0 interface was not there before start the test, let's re-run to check instabilities.

File: automation/test.sh [hunk 3]
```diff
@@ -16,4 +16,9 @@ trap teardown EXIT SIGINT SIGTERM SIGSTOP
 make cluster-down
 make cluster-up
 make cluster-sync
-make E2E_TEST_EXTRA_ARGS="-ginkgo.noColor" test/e2e
+test_args="-ginkgo.v -ginkgo.noColor -test.timeout 20m"
+skip_tests=""
+if [[ $KUBEVIRT_PROVIDER =~ oc- ]]; then
```
Comment (phoracek) [inline automation/test.sh]: please explain why do we do this and when we can drop it

**Comment by phoracek** [comment]
@qinqon drop OS completely?

**Comment by qinqon** [comment]
@phoracek k8s is failing now

**Comment by phoracek** [comment]
@qinqon please post new changes in new commits, Petr is too lazy to look for what has changed :grin:

**Comment by qinqon** [comment]
@phoracek you can clieck "foce pushed" so you get the changes only

https://github.com/nmstate/kubernetes-nmstate/compare/229cfe6c086159f8363fed3940007e001ef595de..28fd013e8852e990d7850d72844b750bebbc3d86

**Comment by phoracek** [comment]
@qinqon whaaaaat! Thanks.

**Comment by phoracek** [comment]
Passed, nice, merging.

**Pull request merged**